subprojects/docs/src/docs/userguide/running-builds/introduction/build_file_basics.adoc

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[build_file_basics]]
+= Build File Basics
+
+Every project has a build script that details build configuration, tasks, and plugins.
+
+image::gradle-basic-4.png[]
+
+Every Gradle build comprises one or more projects; a root project and subprojects.
+Each one has a _build script_.
+
+For single-project builds, the build file contains dependencies and repository information for the build and the project.
+
+For multi-project builds, each subproject has its own build file, and each subproject will be evaluated by executing its build file.
+
+In the build file, two types of dependencies can be added:
+
+1. The libraries and/or plugins on which Gradle and the build script depend.
+2. The libraries on which the project sources depend.
+
+[[sec:build_script]]
+== Build scripts
+
+The build script is either a `build.gradle` file written in Groovy or a `build.gradle.kts` file written in Kotlin.
+
+The link:{groovyDslPath}/index.html[Groovy DSL^] and the link:{kotlinDslPath}/index.html[Kotlin DSL^] are the only accepted languages for Gradle scripts.
+
+Let's take a look at an example and break it down:
+
+====
+[.multi-language-sample]
+=====
+.build.gradle.kts
+[source,kotlin]
+----
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.9.0"                      // <1>
+}
+
+repositories {
+    mavenCentral()                                                      // <2>
+}
+
+dependencies {
+    testImplementation("org.jetbrains.kotlin:kotlin-test-junit5")       // <3>
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.9.3")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    implementation("com.google.guava:guava:32.1.1-jre")
+}
+
+application {
+    mainClass.set("main")                                               // <4>
+}
+
+tasks.named<Test>("test") {
+    useJUnitPlatform()                                                  // <5>
+}
+----
+<1> Add plugins.
+<2> Add dependency location.
```

Comment:
You use the wording `dependency location` here and in the settings file docs. Is there a reason you're not using `repository` (e.g `Add dependency locations` vs `Add repositories`)